### ISSUE TITLE
2024-08-15, Daily Papers for aerodynamics, boundary layer seperation

### ISSUE THREAD

**Issue by 9tailwolf** [9tailwolf/pygithub#53]
# Daily Arxiv Papers

(1) [A systematic dataset generation technique applied to data-driven automotive aerodynamics](http://arxiv.org/abs/2408.07318v1), 2024-08-14
<details>
<summary>Abstract : </summary>
A novel strategy for generating datasets is developed within the context of
drag prediction for automotive geometries using neural networks. A primary
challenge in this space is constructing a training databse of sufficient size
and diversity. Our method relies on a small number of starting data points, and
provides a recipe to interpolate systematically between them, generating an
arbitrary number of samples at the desired quality. We test this strategy using
a realistic automotive geometry, and demonstrate that convolutional neural
networks perform exceedingly well at predicting drag coefficients and surface
pressures. Promising results are obtained in testing extrapolation performance.
Our method can be applied to other problems of aerodynamic shape optimization.
</details>

(2) [High-order projection-based upwind method for simulation of transitional turbulent flows](http://arxiv.org/abs/2408.06698v1), 2024-08-13
<details>
<summary>Abstract : </summary>
We present a scalable, high-order implicit large-eddy simulation (ILES)
approach for incompressible transitional flows. This method employs the
mass-conserving mixed stress (MCS) method for discretizing the Navier-Stokes
equations. The MCS method's low dissipation characteristics, combined with the
introduced operator-splitting solution technique, result in a high-order solver
optimized for efficient and parallel computation of under-resolved turbulent
flows. We further enhance the inherent capabilities of the ILES model by
incorporating high-order upwind fluxes and are examining its approximation
behaviour in transitional aerodynamic flow problems. In this study, we use
flows over the Eppler 387 airfoil at Reynolds numbers up to $3 \cdot 10^5$ as
benchmarks for our simulations.
</details>

(3) [Peaking into the Black-box: Prediction Intervals Give Insight into Data-driven Quadrotor Model Reliability](http://dx.doi.org/10.2514/6.2023-0125), 2024-08-12
<details>
<summary>Abstract : </summary>
Ensuring the reliability and validity of data-driven quadrotor model
predictions is essential for their accepted and practical use. This is
especially true for grey- and black-box models wherein the mapping of inputs to
predictions is not transparent and subsequent reliability notoriously difficult
to ascertain. Nonetheless, such techniques are frequently and successfully used
to identify quadrotor models. Prediction intervals (PIs) may be employed to
provide insight into the consistency and accuracy of model predictions. This
paper estimates such PIs for polynomial and Artificial Neural Network (ANN)
quadrotor aerodynamic models. Two existing ANN PI estimation techniques - the
bootstrap method and the quality driven method - are validated numerically for
quadrotor aerodynamic models using an existing high-fidelity quadrotor
simulation. Quadrotor aerodynamic models are then identified on real quadrotor
flight data to demonstrate their utility and explore their sensitivity to model
interpolation and extrapolation. It is found that the ANN-based PIs widen
considerably when extrapolating and remain constant, or shrink, when
interpolating. While this behaviour also occurs for the polynomial PIs, it is
of lower magnitude. The estimated PIs establish probabilistic bounds within
which the quadrotor model outputs will likely lie, subject to modelling and
measurement uncertainties that are reflected through the PI widths.
</details>

(4) [Towards aerodynamic surrogate modeling based on $β$-variational autoencoders](http://arxiv.org/abs/2408.04969v1), 2024-08-09
<details>
<summary>Abstract : </summary>
Surrogate models combining dimensionality reduction and regression techniques
are essential to reduce the need for costly high-fidelity CFD data. New
approaches using $\beta$-Variational Autoencoder ($\beta$-VAE) architectures
have shown promise in obtaining high-quality low-dimensional representations of
high-dimensional flow data while enabling physical interpretation of their
latent spaces. We propose a surrogate model based on latent space regression to
predict pressure distributions on a transonic wing given the flight conditions:
Mach number and angle of attack. The $\beta$-VAE model, enhanced with Principal
Component Analysis (PCA), maps high-dimensional data to a low-dimensional
latent space, showing a direct correlation with flight conditions.
Regularization through $\beta$ requires careful tuning to improve the overall
performance, while PCA pre-processing aids in constructing an effective latent
space, improving autoencoder training and performance. Gaussian Process
Regression is used to predict latent space variables from flight conditions,
showing robust behavior independent of $\beta$, and the decoder reconstructs
the high-dimensional pressure field data. This pipeline provides insight into
unexplored flight conditions. Additionally, a fine-tuning process of the
decoder further refines the model, reducing dependency on $\beta$ and enhancing
accuracy. The structured latent space, robust regression performance, and
significant improvements from fine-tuning collectively create a highly accurate
and efficient surrogate model. Our methodology demonstrates the effectiveness
of $\beta$-VAEs for aerodynamic surrogate modeling, offering a rapid,
cost-effective, and reliable alternative for aerodynamic data prediction.
</details>

(5) [The effect of Reynolds number on the separated flow over a low-aspect-ratio wing](http://arxiv.org/abs/2408.04801v1), 2024-08-09
<details>
<summary>Abstract : </summary>
At high incidence, low-aspect-ratio wings present a unique set of aerodynamic
characteristics, including flow separation, vortex shedding, and unsteady force
production. Furthermore, low-aspect ratio wings exhibit a highly impactful tip
vortex, which introduces strong spanwise gradients into an already complex
flow. In this work, we explore the interaction between leading edge flow
separation and a strong, persistent tip vortex over a Reynolds number range of
$600 \leq Re \leq 10,000$. In performing this study, we aim to bridge the
insight gained from existing low Reynolds number studies of separated flow on
finite wings ($Re \approx 10^2$) and turbulent flows at higher Reynolds numbers
($Re \approx 10^4$). Our study suggests two primary effects of Reynolds number.
First, we observe a break from periodicity, along with a dramatic increase in
the intensity and concentration of small-scale eddies, as we shift from $Re =
600$ to $Re = 2,500$. Second, we observe that many of our flow diagnostics,
including the time-averaged aerodynamic force, exhibit reduced sensitivity to
Reynolds number beyond $Re = 2,500$, an observation attributed to the
stabilizing impact of the wing tip vortex. This latter point illustrates the
manner by which the tip vortex drives flow over low-aspect-ratio wings, and
provides insight into how our existing understanding of this flowfield may be
adjusted for higher Reynolds number applications.
</details>

(6) [On the unsteady aerodynamics of flapping wings under dynamic hovering kinematics](http://dx.doi.org/10.1063/5.0215531), 2024-08-06
<details>
<summary>Abstract : </summary>
Hummingbirds and insects achieve outstanding flight performance by adapting
their flapping motion to the flight requirements. Their wing kinematics can
change from smooth flapping to highly dynamic waveforms, generating unsteady
aerodynamic phenomena such as leading-edge vortices (LEV), rotational
circulation, wing wake capture, and added mass. This article uncovers the
interactions of these mechanisms in the case of a rigid semi-elliptical wing
undergoing aggressive kinematics in the hovering regime at $Re\sim
\mathcal{O}(10^3)$. The flapping kinematics were parametrized using smoothed
steps and triangular functions and the flow dynamics were simulated by
combining the overset method with Large Eddy Simulations (LES). The analysis of
the results identifies an initial acceleration phase and a cruising phase.
During the former, the flow is mostly irrotational and governed by the added
mass effect. The added mass was shown to be responsible for a lift first peak
due to the strong flapping acceleration. The dynamic pitching and the wing wake
interaction generate a second lift peak due to a downwash flow and a vortex
system on the proximal and distal parts of the wing's pressure side.
Conversely, aerodynamic forces in the cruising phase are mainly governed by the
growth and the establishment of the LEV. Finally, the leading flow structures
in each phase and their impact on the aerodynamic forces were isolated using
the extended Proper Orthogonal Decomposition (POD).
</details>

(7) [A unified transition mechanism from shock to detonation waves](http://arxiv.org/abs/2408.03042v1), 2024-08-06
<details>
<summary>Abstract : </summary>
The transition of shock-to-detonation is of great significance for the
investigation of supernova formation, disaster prevention and supersonic
propulsion technology. In this paper, the influence Equation of
shock-to-detonation transition is summarized for the oblique detonation problem
from aerodynamic analysis. The Equation integrates the effects of parameters
such as chemical reaction, shock intensity and wall conditions, which
quantitatively explains the physical mechanism of shock-to-detonation
transition in the form of mathematical expression. Comparison with numerical
simulation results as well as their gradients verified the reliability of the
influence Equation. Further, the influence Equation can also be used to predict
the critical conditions for the transition from shock to detonation transition
form. In addition to oblique detonation, the influence Equation is compatible
with the deflagration-to-detonation problem for normal detonation, which shows
a wide applicability.
</details>

(8) [AhmedML: High-Fidelity Computational Fluid Dynamics Dataset for Incompressible, Low-Speed Bluff Body Aerodynamics](http://arxiv.org/abs/2407.20801v1), 2024-07-30
<details>
<summary>Abstract : </summary>
The development of Machine Learning (ML) methods for Computational Fluid
Dynamics (CFD) is currently limited by the lack of openly available training
data. This paper presents a new open-source dataset comprising of high
fidelity, scale-resolving CFD simulations of 500 geometric variations of the
Ahmed Car Body - a simplified car-like shape that exhibits many of the flow
topologies that are present on bluff bodies such as road vehicles. The dataset
contains simulation results that exhibit a broad set of fundamental flow
physics such as geometry and pressure-induced flow separation as well as 3D
vortical structures. Each variation of the Ahmed car body were run using a
high-fidelity, time-accurate, hybrid Reynolds-Averaged Navier-Stokes (RANS) -
Large-Eddy Simulation (LES) turbulence modelling approach using the open-source
CFD code OpenFOAM. The dataset contains boundary, volume, geometry, and
time-averaged forces/moments in widely used open-source formats. In addition,
the OpenFOAM case setup is provided so that others can reproduce or extend the
dataset. This represents to the authors knowledge, the first open-source
large-scale dataset using high-fidelity CFD methods for the widely used Ahmed
car body that is available to freely download with a permissive license
(CC-BY-SA).
</details>

(9) [Large-Scale Horizontal Axis Wind Turbine Aerodynamic Efficiency Optimization using Active Flow Control and Synthetic Jets](http://arxiv.org/abs/2407.20746v1), 2024-07-30
<details>
<summary>Abstract : </summary>
Efficiency increase is seen as one of the main goals in any energy converting
device. In this direction, the present study aims to demonstrate that
large-scale wind turbines can still be improved in order to generate larger
amounts of energy. The research presented in this manuscript consists of two
main blocks. Initially, it analyzes via Computational Fluid Dynamics (CFD) the
boundary layer dynamics on a set of pre-determined airfoils cut along the
DTU-10MW reference blade by using the 2D URANS k omega SST turbulence model.
The aim of this initial stage is to identify the boundary layer separation
point, its associated frequency, and peak to peak amplitude for each airfoil
cut along the blade, evaluating as well their respective aerodynamic
characteristics. The second main goal of the present research consists of
implementing the Active Flow Control (AFC) technology and, when employing
synthetic jets, to reattach the boundary layer in all airfoils where it is
separated. To accomplish this second goal efficiently, the five parameters
associated to the AFC implementation performed to each airfoil will be obtained
through respective genetic algorithm optimizations. An energy assessment is
finally undertaken at each airfoil to validate the respective energy gain
obtained. When comparing the net power gain, before and after AFC
implementation, generated by each of the airfoils evaluated, net power gains
between 23 and 36KW are obtained in all airfoils analyzed, clarifying that the
proposed technology is capable of improving the performance of the wind
turbines, very likely at any operating condition.
</details>

(10) [Aero-Nef: Neural Fields for Rapid Aircraft Aerodynamics Simulations](http://arxiv.org/abs/2407.19916v1), 2024-07-29
<details>
<summary>Abstract : </summary>
This paper presents a methodology to learn surrogate models of steady state
fluid dynamics simulations on meshed domains, based on Implicit Neural
Representations (INRs). The proposed models can be applied directly to
unstructured domains for different flow conditions, handle non-parametric 3D
geometric variations, and generalize to unseen shapes at test time. The
coordinate-based formulation naturally leads to robustness with respect to
discretization, allowing an excellent trade-off between computational cost
(memory footprint and training time) and accuracy. The method is demonstrated
on two industrially relevant applications: a RANS dataset of the
two-dimensional compressible flow over a transonic airfoil and a dataset of the
surface pressure distribution over 3D wings, including shape, inflow condition,
and control surface deflection variations. On the considered test cases, our
approach achieves a more than three times lower test error and significantly
improves generalization error on unseen geometries compared to state-of-the-art
Graph Neural Network architectures. Remarkably, the method can perform
inference five order of magnitude faster than the high fidelity solver on the
RANS transonic airfoil dataset. Code is available at
https://gitlab.isae-supaero.fr/gi.catalani/aero-nepf
</details>

(11) [WindsorML: High-Fidelity Computational Fluid Dynamics Dataset For Automotive Aerodynamics](http://arxiv.org/abs/2407.19320v2), 2024-07-27
<details>
<summary>Abstract : </summary>
This paper presents a new open-source high-fidelity dataset for Machine
Learning (ML) containing 355 geometric variants of the Windsor body, to help
the development and testing of ML surrogate models for external automotive
aerodynamics. Each Computational Fluid Dynamics (CFD) simulation was run with a
GPU-native high-fidelity Wall-Modeled Large-Eddy Simulations (WMLES) using a
Cartesian immersed-boundary method using more than 280M cells to ensure the
greatest possible accuracy. The dataset contains geometry variants that
exhibits a wide range of flow characteristics that are representative of those
observed on road-cars. The dataset itself contains the 3D time-averaged volume
& boundary data as well as the geometry and force & moment coefficients. This
paper discusses the validation of the underlying CFD methods as well as
contents and structure of the dataset. To the authors knowledge, this
represents the first, large-scale high-fidelity CFD dataset for the Windsor
body with a permissive open-source license (CC-BY-SA).
</details>

(12) [Numerical simulations of attachment-line boundary layer in hypersonic flow, Part I: roughness-induced subcritical transitions](http://arxiv.org/abs/2407.15465v1), 2024-07-22
<details>
<summary>Abstract : </summary>
The attachment-line boundary layer is critical in hypersonic flows because of
its significant impact on heat transfer and aerodynamic performance. In this
study, high-fidelity numerical simulations are conducted to analyze the
subcritical roughness-induced laminar-turbulent transition at the leading-edge
attachment-line boundary layer of a blunt swept body under hypersonic
conditions. This simulation represents a significant advancement by
successfully reproducing the complete leading-edge contamination process
induced by surface roughness elements in a realistic configuration, thereby
providing previously unattainable insights. Two roughness elements of different
heights are examined. For the lower-height roughness element, additional
unsteady perturbations are required to trigger a transition in the wake,
suggesting that the flow field around the roughness element acts as a
disturbance amplifier for upstream perturbations. Conversely, a higher
roughness element can independently induce the transition. A low-frequency
absolute instability is detected behind the roughness, leading to the formation
of streaks. The secondary instabilities of these streaks are identified as the
direct cause of the final transition.
</details>

(13) [Investigation of discontinuous Galerkin methods in adjoint gradient-based aerodynamic shape optimization](http://arxiv.org/abs/2407.18963v1), 2024-07-18
<details>
<summary>Abstract : </summary>
This work develops a robust and efficient framework of the adjoint
gradient-based aerodynamic shape optimization (ASO) using high-order
discontinuous Galerkin methods (DGMs) as the CFD solver. The adjoint-enabled
gradients based on different CFD solvers or solution representations are
derived in detail, and the potential advantage of DG representations is
discovered that the adjoint gradient computed by the DGMs contains a
modification term which implies information of higher-order moments of the
solution as compared with finite volume methods (FVMs). A number of numerical
cases are tested for investigating the impact of different CFD solvers
(including DGMs and FVMs) on the evaluation of the adjoint-enabled gradients.
The numerical results demonstrate that the DGMs can provide more precise
adjoint gradients even on a coarse mesh as compared with the FVMs under coequal
computational costs, and extend the capability to explore the design space,
further leading to acquiring the aerodynamic shapes with more superior
aerodynamic performance.
</details>

(14) [Droplet breakup and size distribution in an airstream -- effect of inertia](http://arxiv.org/abs/2407.13160v1), 2024-07-18
<details>
<summary>Abstract : </summary>
We experimentally investigate the morphology and breakup of a droplet as it
descends freely from a height and encounters an airstream. The size
distributions of the child droplets are analysed using high-speed shadowgraphy
and in-line holography techniques. We found that a droplet falling from various
heights exhibits shape oscillations due to the intricate interplay between
inertia and surface tension forces, leading to significant variations in the
radial deformation of the droplet, influencing the breakup dynamics under an
identical airstream condition. Specifically, the droplet undergoes vibrational
breakup when introduced at a location slightly above the air nozzle. In
contrast, as the release height of the droplet increases, keeping the Weber
number defined based on the velocity of the airstream fixed, a dynamic
interplay between the inertia of the droplet and the aerodynamic flow field
comes into play, resulting in a sequence of breakup modes transitioning from
vibrational breakup to retracting bag breakup, bag breakup, bag-stamen,
retracting bag-stamen breakup, and eventually returning to vibrational breakup.
Our experiments also reveal that the size distribution resulting from
retracting bag breakup primarily arises from rim and node fragmentation,
leading to a bimodal distribution. In contrast, bag and bag-stamen breakups
yield a tri-modal size distribution due to the combined contributions of bag,
rim, and node breakup mechanisms. Furthermore, we utilize a theoretical model
that incorporates the effective Weber number, considering different release
heights. This model accurately predicts the size distribution of the child
droplets resulting from the various breakup modes observed in our experiments.
</details>

(15) [Wind Tunnel Testing and Modeling Implications of an Advanced Turbine Cascade](http://arxiv.org/abs/2407.11210v1), 2024-07-15
<details>
<summary>Abstract : </summary>
This paper describes the extensive Wind Tunnel (WT) linear cascade testing
campaign carried out on a constant section turbine blade developed for low
subsonic applications. Comprehensive experimental program was designed to
determine the aerodynamic behaviour of this blade under a wide range of varying
geometrical like Pitch to Chord Ratio, Stagger Angle and Aerodynamic like
Reynolds Roughness, Mach number, Incidence angle conditions. In addition to the
classical Two-Dimensional (2D) measurements, targeted Three-Dimensional (3D)
surveys have been performed to complement the 2D results, allowing to draw
supplementary conclusions as to the behaviour of the considered blade in
three-dimensions. The performed experiments predominantly covered the
transitional and beginning of a fully turbulent flow regime (maximum Re is 2.5
million). Post processing of experimental results were done in view of
exploitation and judgment of some key aerodynamic aspects; parameters such as
profile section load distribution, loss coefficient and flow deviation angle.
Alongside with confirmation of the design targets, a parallel goal pursued in
the current WT testing, was to get a more accurate idea as to the predictive
capabilities of the employed CFD tools (CFX and MISES) under the basic flow
condition generated in the WT. The aim was to get a global picture, as to the
level of agreement that can be reached by the used design tools while
accounting for the afore-mentioned design, operational variables. The obtained
measurement results clearly indicated the success of the design in achieving
the pre-set targets. The paper further provides a detailed discussion on the
identified discrepancies, measurement versus predictions, of the key parameters
at the geometric, aerodynamic design as well as off-design conditions.
</details>
